scripts/distribution/ubuntu-packages/template/debian/concordium-node.dirs

```diff
@@ -0,0 +1,2 @@
+/var/lib/concordium/b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d/config
+/var/lib/concordium/b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d/data
```

Comment:
The data dir will be created because the genesis.dat file is copied into it, so it should not be here.

Also, please add this file to one of the things that needs to be updated when building the package so that it is not forgotten.